<?xml version="1.0"?>
<launch>
  <!-- <param name="robot_name" value="VS087A4-AV6-NNN-NNN A"/> -->
  <param name="robot_name" value="vs087"/>
  <param name="joint_1" value="1"/>
  <param name="joint_2" value="1"/>
  <param name="joint_3" value="1"/>
  <param name="joint_4" value="1"/>
  <param name="joint_5" value="1"/>
  <param name="joint_6" value="1"/>
  <param name="arm_group" value="0"/>
  <param name="finger_3rd_joint" value="0"/>
  <param name="finger_L_joint" value="1"/>
  <param name="finger_R_joint" value="1"/>
  <param name="gripper_group" value="0"/>
</launch>
